core/templates/services/validators.service.spec.ts

```diff
@@ -24,26 +24,23 @@ import { ValidatorsService } from 'services/validators.service';
 
 describe('Validators service', () => {
   let vs: ValidatorsService = null;
-  /* eslint-disable dot-notation */
   const INVALID_NAME_CHARS_COPY = (
+    /* eslint-disable-next-line dot-notation */
```

Comment:
Couldn't the next line just be AppConstants.INVALID_NAME_CHARS, and similarly elsewhere?

Reply:
Seems like someone else fixed it already